packages/manager-components/src/enumTypes/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './locale';
+export * from './intervalUnit';
```

Comment:
Is it exported by the index of the lib ? (available from import '@ovhcloud/manager-components')